package-lock.json

```diff
@@ -51,7 +51,7 @@
 		},
 		"apps/backend": {
 			"name": "@meetlytic/backend",
-			"version": "1.1.0",
+			"version": "1.3.0",
```

Comment:
Your package.json file says the version is 1.4.0, but package-lock says 1.3.0, this is not correct